### PR TITLE
Updates for Multi_Fit notebook + Periodogram

### PR DESCRIPTION
**Pull request by pearsonkyle**

### CODE REVIEW

File: examples/tess.py [hunk 1]
```diff
@@ -739,27 +739,8 @@ def check_std(time, flux, dt=0.5): # dt = [hr]
 
         # create AAVSO formatted csv
         csv_lk = OutputFiles(myfit, prior, infoDict, planetdir)
-        try:
-            csv_lk.aavso_csv(airmass,u0,u1,u2,u3,tmidstr)
-            csv_lk.aavso(airmass,u0,u1,u2,u3, tmidstr)
-        except:
-            print(f"Failed to create AAVSO csv for {args.target} - Sector {lcdata['sector']}")
+        csv_lk.aavso_csv(airmass,u0,u1,u2,u3,tmidstr)
+        csv_lk.aavso(airmass,u0,u1,u2,u3, tmidstr)
```
Comment (rzellem) [inline examples/tess.py:743]: any reason to remove this try/except, @pearsonkyle ?

File: exotic/api/nested_linear_fitter.py [hunk 2]
```diff
@@ -333,11 +335,12 @@ def plot_periodogram(self):
 
         ########################################
         # subtract first order solution from data and recompute periodogram
-        maxper = 50 # constrain second order solution to be less than 50 days
+        maxper = maxper2 # constrain second order solution to be less than 50 days
```
Comment (rzellem) [inline exotic/api/nested_linear_fitter.py:338]: should this comment be updated?

File: exotic/api/output_aavso.py [hunk 3]
```diff
@@ -112,7 +112,6 @@ def aavso(self, airmasses, ld0, ld1, ld2, ld3, tmidstr):
                     f"#PRIORS-XC={dumps(priors_dict)}\n"  # code yields
                     f"#RESULTS=Tc={round_to_2(self.fit.parameters['tmid'], self.fit.errors['tmid'])} +/- {round_to_2(self.fit.errors['tmid'])}"
                     f",Rp/R*={round_to_2(self.fit.parameters['rprs'], self.fit.errors['rprs'])} +/- {round_to_2(self.fit.errors['rprs'])}"
-                    f",a/R*={round_to_2(self.fit.parameters['ars'], self.fit.errors['ars'])} +/- {round_to_2(self.fit.errors['ars'])}"
```
Comment (rzellem) [inline exotic/api/output_aavso.py:115]: why did you remove a/Rs here? because we don't fit for it explicitly?

**Review by rzellem — CHANGES_REQUESTED** [review]

File: examples/tess.py (hunk 1 above)
Comment (pearsonkyle) [reply examples/tess.py:743]: It is not necessary and complicates debugging if you can't get a traceback on the error. The errors have been fixed and the code runs without it

File: exotic/api/output_aavso.py (hunk 3 above)
Comment (pearsonkyle) [reply exotic/api/output_aavso.py:115]: Yes, a/Rs is not always a free parameter and this line of code will error out. I've switched to fitting for inclination in order to estimate the transit duration

Comment (pearsonkyle) [reply exotic/api/output_aavso.py:115]: this line was causing that try/except statement above to trigger the except due to my comment above. It's not necessary to include it since we already list the value in the priors.

File: exotic/api/nested_linear_fitter.py (hunk 2 above)
Comment (pearsonkyle) [reply exotic/api/nested_linear_fitter.py]: removed the comment... the period grid should likely be hand tailored anyway based on need.. it's tough to choose a one size fits all range

**Review by rzellem — APPROVED** [review]
